src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -18,6 +18,9 @@
  * @author Kohsuke Kawaguchi
  */
 public class GHOrganization extends GHPerson {
+
+    private boolean has_organization_projects;
```

Comment:
https://docs.github.com/en/rest/reference/orgs#get-an-organization 
This shows two different projects fields. Do you know what the difference is? 

```
  "has_organization_projects": true,
  "has_repository_projects": true,
```

Reply:
![image](https://user-images.githubusercontent.com/14114123/117916197-40f0a880-b319-11eb-8c4d-9bc70b941918.png)

From the docs.